jcl/src/jdk.management/share/classes/com/ibm/lang/management/RuntimeMXBean.java

```diff
@@ -83,7 +83,15 @@ public String idleStateName() {
 	 * 
 	 * @return A long representing the process ID (pid) on the underlying 
 	 * operating system.
+	/*[IF Java10]
+	 * 
+	 * @deprecated(forRemoval = true, since = "10") Use 
+	 * {@link java.lang.management.RuntimeMXBean#getPid() getPid()} instead.
+	 */
+	@Deprecated
```

Comment:
This should be @Deprecated(forRemoval=true, since="10"). See java.lang.Class.newInstance() for an example.